RubeesEat.IntegrationTests/DatabaseIntegrationTestBase.cs

```diff
@@ -0,0 +1,46 @@
+using System.Data;
+using Microsoft.Extensions.Configuration;
+using RubeesEat.Model.DB;
+
+namespace RubeesEat.IntegrationTests;
+
+[Explicit("does not run on teamcity")]
+public abstract class DatabaseIntegrationTestBase
+{
+    protected SqlConnectionFactory Factory;
+    private readonly string _connectionStringName = "DefaultConnectionString";
+
+    [SetUp]
+    public void SetUp()
+    {
+        string filePath = "./CreateTestDb.sql";
+        string sqlContent = File.ReadAllText(filePath);
+        string[] sqlCommands = sqlContent.Split([';'], StringSplitOptions.RemoveEmptyEntries);
+
+        // Trimmt leere Zeilen oder unnötige Leerzeichen von jedem Befehl
```

Comment:
We can remove the obvious comment. There are cases where a comment is necessary to explain what is happening but a loop with 1 statement should be obvious enough for the comment to be unnecessary. Regarding comments, they are usually more necessary to explain why something is happening. If they explain what is happening, they are in danger of becoming out of date in the future when the next person forgets to update them.